blog-posts/boosting-developer-productivity-with-github-actions.md

```diff
@@ -0,0 +1,266 @@
+---
+path: "/blog/boosting-developer-productivity-with-github-actions/"
+date: "2022-02-25"
+title: Boosting developer productivity with GitHub Actions
+featuredImage: images/github-actions/hero.jpg
```

Comment:
I was aware of this but wasn't able to find a nice compromise between looking-good-on-desktop and looking-good-in-preview and looking-good-on-mobile. I found an aspect ratio and image size that kind of works... but am absolutely open for ideas :pray:

Reply:
Yeah it's difficult to align text. I had the same problem with the metaverse post. I suggest we keep it as is because it's good enough and then check if we can introduce either a mechanism to influence the cropping or directly provide two different images, one for the header and one for the teasers (which will still be resized/processed anyway).